o8g/Sets/The Path to Carcosa/set.xml

```diff
@@ -937,6 +937,7 @@
     <property name="Encounter Set" value="Curtain Call"/>
     <property name="Unique" value="ο"/>
     <property name="Type" value="Enemy"/>
+	<property name="Subtype" value="Weakness"/>
```

Comment:
Just a nitpick here, you'd want another space character at the beginning of the line to align it with the other lines 😄

Reply:
God i hate spaces. Fixed